exla/test/test_helper.exs

```diff
@@ -1,6 +1,10 @@
 target = System.get_env("EXLA_TARGET", "host")
 client = EXLAHelpers.client()
 
+if System.get_env("DEBUG") in ["1", "true"] do
+  IO.gets("Press enter to continue... -- PID: #{System.pid()}")
+end
```

Comment:
Can we leave this in this time? Or something similar? Basically whenever I need to debug with lldb this is necessary.

Reply:
Yes, leave it :)